index.html

```diff
@@ -26,48 +38,54 @@ <h2>I’m Mariane <br />Glad to see you!</h2>
             you like what you see and have a project you need coded, don’t
             hestiate to contact me.
           </p>
-        </div>
-        <div class="social-media">
-          <div class="icon-a">
-            <h5><a href="#">Let's connect</a></h5>
+          <div class="social-media">
+            <div class="icon-a">
+              <h5><a href="#">Let's connect</a></h5>
+            </div>
+            <ul>
```

Comment:
- [ ] Kindly make sure, as required by the previous reviewer, to make the **Let's Connect** sub-title math the Gigma styling, like on Figma it on **Uppercase** and yours still in lowercase.

Reply:
Udated Let's Connect to be UPPERCASE

Reply:
This issue is well resolved! ✔️